subprojects/settings.gradle.kts

```diff
@@ -174,13 +195,32 @@ pluginManagement {
     }
 }
 
+val artifactoryUrl: String? by settings
+
+fun MavenArtifactRepository.artifactoryUrl(repositoryName: String) {
+    setUrl("$artifactoryUrl/$repositoryName")
+    isAllowInsecureProtocol = true
+}
+
+fun MavenArtifactRepository.setUrlOrProxy(repositoryName: String, originalRepo: String) {
+    if (artifactoryUrl.isNullOrBlank()) {
```

Comment:
How should I know about this property?

Let's add it to the documentation. I see it as a part of "Contibuting / How to start"